packages/tfm-ror52/tfm-ror52.spec

```diff
@@ -121,13 +122,16 @@ ruby -rfileutils > rubygems_filesystem.list << \EOR
   FileUtils.mkdir_p File.join '%{buildroot}', Gem.default_ext_dir_for('%{gem_dir}')
 
   # Output the relevant directories.
-  puts Gem.default_dirs['%{scl}_system'.to_sym].values
+  Gem.default_dirs['%{scl}_system'.to_sym].values
```

Comment:
Was the puts no longer needed? How does this otherwise output it?

Reply:
This is more inline with the documentation from RHSCL packaging guide now.